packages/runtime/container-runtime/src/summary/summaryFormats.md

```diff
@@ -0,0 +1,150 @@
+# Summary and snapshot formats
```

Comment:
Consider adding a table of contents, like we have in opLifecycle/Readme.md

Reply:
Good suggestion. Added a table of contents.